packages/core/content-releases/server/src/controllers/release.ts

```diff
@@ -36,20 +38,31 @@ const releaseController = {
     });
 
     await permissionsManager.validateQuery(ctx.query);
-    const query = await permissionsManager.sanitizeQuery(ctx.query);
 
-    const isPaginatedRequest =
-      query && Object.keys(query).some((key) => ['page', 'pageSize'].includes(key));
+    const releaseService = getService('release', { strapi });
 
-    if (isPaginatedRequest) {
-      const { results, pagination } = await getService('release', { strapi }).findPage(query);
-      // Format the data object
-      const data = formatDataObject(results);
+    // Handle requests for releases filtered by content type entry
+    const isFindManyForContentTypeEntry = Boolean(ctx.query?.contentTypeUid && ctx.query?.entryId);
+    if (isFindManyForContentTypeEntry) {
+      const query: GetContentTypeEntryReleases.Request['query'] =
+        await permissionsManager.sanitizeQuery(ctx.query);
+
+      const contentTypeUid = query.contentTypeUid;
+      const entryId = query.entryId;
+      // Parse the string value or fallback to a default
+      const hasEntryAttached: GetContentTypeEntryReleases.Request['query']['hasEntryAttached'] =
+        typeof query.hasEntryAttached === 'string' ? JSON.parse(query.hasEntryAttached) : true;
```

Comment:
This is more personal, but let me know what you think. I would prefer this to fallback to false. If we don't pass `hasEntryAttached` I expect this to be false because is not present